terraform/modules/paas/main.tf

```diff
@@ -90,6 +90,33 @@ resource cloudfoundry_route web_app_find_gov_uk_route {
   hostname = each.value
 }
 
+locals {
+  target_app = var.find_route_target == "find" ? data.cloudfoundry_app.find_app.id : resource.cloudfoundry_app.web_app.id
+}
+
+resource cloudfoundry_route find_web_app_cloudapps_digital_route {
+  count = local.find_cloudapp_names != null ? 1 : 0
+
+  domain   = data.cloudfoundry_domain.london_cloudapps_digital.id
+  space    = data.cloudfoundry_space.space.id
+  hostname = local.find_cloudapp_names
```

Comment:
This can use the constant `local.find_app_name` instead

Reply:
changed